scenes/PlanetSurface/PlanetSurface.gd

```diff
@@ -10,9 +10,6 @@ extends CanvasLayer
 func _ready():
 	pass # Replace with function body.
 
-func _input(event) -> void:
-	if event.is_action_pressed("take_off"):
-		GameState.scene = Constants.SceneId.World
-# Called every frame. 'delta' is the elapsed time since the previous frame.
-#func _process(delta):
-#	pass
+
+func _on_Leave_pressed():
```

Comment:
Do we still want to support the take_off keyboard shortcut? If not we should remove it from the key mappings. If so we should keep the _input function as well. My vote would be to keep it, but maybe change it to the L key for "leave" (and it also matches L for land so you use the same key to change to and from the scene. What do you guys think?